Vagrantfile

```diff
@@ -55,8 +55,8 @@ ec2_iam_instance_profile_name = nil
 
 ebs_volume_type = 'gp3'
 
-jdk_major = '8'
-jdk_full = '8u202-linux-x64'
+jdk_major = '11'
+jdk_full = '11.0.2-linux-x64'
```

Comment:
Temp fix until #17576 gets merged - I'll revert this if the tests pass and I can merge